src/Documentation/Action/EntrypointAction.php

```diff
@@ -37,13 +37,13 @@ public function __construct(
         private readonly ResourceNameCollectionFactoryInterface $resourceNameCollectionFactory,
```

Comment:
By the way, this class should probably be undeprecated as it is now used again.

Reply:
Shouldn't we just copy that class to the Laravel directory?

Reply:
As you wish